### DISCUSSION TITLE
1.3.0-canary.7 – Oct 28, 2024

### DISCUSSION THREAD

**Discussion by blefnk** [blefnk/relivator#47] [Announcements]
:up: **Relivator 1.3.0-canary.7** && **Reliverse 0.8.0**  :up:

:tada:**It happened, friends! It’s here — the fully updated Relivator!** :tada:

This is *the final canary for 1.3.0*. The next release will be the release candidate, means RC, followed by the long-awaited 1.3.0 release.

- Version 1.3.0 will lay the foundation for an updated roadmap moving forward.
- Relivator now lets you enjoy the refreshed design. And this is just the beginning of the global redesign!
- It’s easier to breathe now. Cleaner things, cleaner code. A lot of clutter is gone. You'll love it!

canary.7 is the first Relivator's build built using the [Reliverse CLIhttps://github.com/reliverse/cli](). While **Reliverse 0.8.0** has already "learned" to do a lot, it’s still in its early stages of development according to the future awesome and huge plans. This means some elements from previous versions of Relivator weren’t built-in into this new canary. Yet the new Relivator continues to work perfectly even without those things.

However, the release version of Relivator 1.3.0 will include all the important missing pieces. If you need a missing file, you can always use the following repository: [blefnk/all-in-one-nextjs-template](https://github.com/blefnk/all-in-one-nextjs-template). By the way, it, like Relivator, can already be easily installed and configured using Reliverse!

And now, the biggest news that everyone’s been waiting for — **the functionality of almost all Relivator pages has been restored**! :champagne::tada:

There are still a few app components that need to be adjusted for Next.js 15, but this will be handled with the RC and 1.3.0 release.

So, there’s still a lot of work ahead. But, dear community, today we celebrate! :muscle:

:drum:  :point_right:  https://relivator.com/en-US

<hr /><em>This discussion was created from the release <a href='https://github.com/blefnk/relivator-nextjs-template/releases/tag/'>1.3.0-canary.7 – Oct 28, 2024</a>.</em>

**Comment by andrepatt** [comment]
Site link not working anymore 🙁

**Reply by blefnk** [reply]
It's alive again! Thanks for letting me know! 😊
